### ISSUE TITLE
Improve explanation of translation workflow in README

### ISSUE THREAD

**Issue by joelnitta** [swcarpentry-ja/i18n#75]
We should clarify that the translation is done on a "per-section" basis for each lesson, and how to find the section within the PO file. The PO files are really long (thousands of lines), so it's daunting to find the right place to edit if you don't know how they are set up.

**Comment by TomKellyGenetics** [comment]
It's explained well here:  https://carpentries-i18n-handbook.readthedocs.io/en/latest/guide/translator/index.html

We could fork this and change it for our purposes?

**Comment by joelnitta** [comment]
Yes, I think forking it and migrating **all** our documentation to such a website would be a good idea. It would help us keep things in sync with the general carpentries i18n approach. We should also have a translation available for such a website too... presumably with PO files?
https://docs.readthedocs.io/en/stable/guides/manage-translations.html

**Comment by TomKellyGenetics** [comment]
Yes that it seems to be possible to support multiple locales with PO files. Need to look into this but seems to be well-documented for how to do this.

https://github.com/readthedocs/readthedocs.org/issues/4315
https://docs.readthedocs.io/en/stable/localization.html
https://docs.readthedocs.io/en/stable/custom_installs/customization.html

For example:
https://docs.phpmyadmin.net/en/latest/
https://docs.phpmyadmin.net/ja/latest/

![Screen Shot 2020-11-24 at 14 02 58](https://user-images.githubusercontent.com/5493325/100050944-ebedd280-2e5d-11eb-9424-73d6fbcde12a.png)
